examples/linear_regression/linear_regression.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
@jcabrero there are several errors in the cell outputs & the text still says `Dot product` instead of `Linear regression`